### DISCUSSION TITLE
w2grid save filter button

### DISCUSSION THREAD

**Discussion by quagliarellamichel** [vitmalina/w2ui#2252] [Q&A] [answered]
hi guys

the buttom of filter's save open the pop-up window , i can write name of the new filter but after that if i select cancel or save teh window remain open.
the filter is saved when we press save but because the window remain open we have to refresh the page to see it and close window.

any solution?

**Comment by vitmalina** [comment, marked as answer]
This should be created as an issue. I briefly looked, it looks like some minification issue

**Comment by vitmalina** [comment]
Fixed. See https://w2ui.com/web/demos/#/grid/16

**Reply by quagliarellamichel** [reply]
Grat! tks
